### ISSUE TITLE
Module shows Miles with a 'km' label with Imperial configuration setting

### ISSUE THREAD

**Issue by jpiscaer** [f00d4tehg0dz/MMM-TeslaFi#1]
I have set imperial: false in the config.
The modules is still showing values in miles, but instead of the 'miles' label, it shows 'km' label.

**Comment by f00d4tehg0dz** [comment]
![image](https://user-images.githubusercontent.com/1635526/67041254-406b3e00-f0f3-11e9-99a7-b41cfc9c025d.png)
![image](https://user-images.githubusercontent.com/1635526/67041278-4e20c380-f0f3-11e9-8609-8cff7c343bae.png)

strange its converting just fine for me. Is there a specific value that needs to be converted that isn't?

**Comment by jpiscaer** [comment]
Strange that you're not seeing it. All values are affected in mine; so let me dive into this on my end and report back.

**Comment by jpiscaer** [comment]
![Screenshot 2019-11-29 at 13 07 22](https://user-images.githubusercontent.com/7115118/69868147-35d2c580-12a9-11ea-8792-da3d0ec976a7.png)
I often see no data, like in the screenshot. I think it correlates to when my car is in deep sleep mode.. Any idea what I can do to fix?

**Comment by f00d4tehg0dz** [comment]
yes an easy way to fix this is in your car enable the settings energy
saving to Off and always connected to On. Then in teslafi.com there is I
believe a setting that polls your car in intervals to wake it from deep
sleep. You can change those settings too.

**Comment by justynr** [comment]
I ran into this too - it seems that Teslafi is reporting Miles no matter what the interface of Teslafi is set to.

I use KM in my car, Teslafi and on Magic Mirror. 

I changed the math in the .js file - I noticed that set to imperial, it was 30,000 something. 
In KM it was 52,000 something.
I actually have 92,000 something.

Changed the divide to multiply, all is working.

**Comment by f00d4tehg0dz** [comment]
Nice Fix! Thank you
